new/LIT7217PPYaqi.xml

```diff
@@ -0,0 +1,87 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7217PPYaqi" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Prayer for expelling budā, Bāryā, demons, devils, and different diseases</title>
```

Comment:
If it is about expelling, I expect Budā to be understood as a demon here. I would suggest to capitalise the term for that reason.